src/image.py

```diff
@@ -260,9 +266,43 @@ def push_image(self):
         self.summary["status"] = constants.STATUS_MESSAGE[self.build_status]
         self.summary["end_time"] = datetime.now()
         self.summary["ecr_url"] = self.ecr_url
+        if "pushed_uris" not in self.summary:
+            self.summary["pushed_uris"] = []
+        self.summary["pushed_uris"].append(f"{self.repository}:{tag}")
+        response.append(f"Completed Push for {self.repository}:{tag}")
         self.log.append(response)
 
         LOGGER.info(f"DOCKER PUSH LOGS: \n {self.get_tail_logs_in_pretty_format(2)}")
-        LOGGER.info(f"Completed Push for {self.name}")
+        return self.build_status
+
+    def push_image_with_additional_tags(self):
```

Comment:
Why do we need at separate function to push a additonal tag? Why cant be this done in `push_image`

Reply:
Seems that we need to tag the docker image first and push to ECR later.